### PR TITLE
fix: get count of all reports when no filter is applied

### PR DESCRIPTION
**Pull request by BryanCaldeira**

### CODE REVIEW

File: assets/models/Map.js [hunk 1]
```diff
@@ -101,6 +101,12 @@ class Map {
 
   filterMarkerCount(subCategoryIdArr = []) {
     let count = 0;
+    
+    // when no filter option is applied give count of all reports
+    if (subCategoryIdArr.length === 0) {
+      return this.mapLayers.getLayers().length;
+    }
+
```
Comment (BryanCaldeira) [inline assets/models/Map.js:109]: when no filter is applied, this gets the count of the markers on the map - only hazard markers

File: pages/home/script.js [hunk 2]
```diff
@@ -482,11 +482,6 @@ const hazardFilterTempApply = async ({ target }) => {
     target.classList.remove('selected');
     // all filters are de-selected
 
-    if (hazardTempFilters.length === 0) {
-      hazardShowCount = 0;
-      showReportsBtnStatus(hazardShowCount);
-      return;
-    }
     hazardShowCount = geoMap.filterMarkerCount(hazardTempFilters);
```
Comment (BryanCaldeira) [inline pages/home/script.js:485]: This logic was no longer needed as we show `all reports` when `hazardTempFilters` array has no id's in it.

**Review by ReinhardtBotha — APPROVED** [review]

**Comment by whamester** [comment]
Clear the filters:
- This should be a two-step action: clear the filters and then apply the changes.
 - Clicking the `Clear All` should clear the selected checkboxes and display the count of the `Show # reports` button, by clicking on `Show # reports` is when the filters are applied on the map.

Styles:
- Double-check the styles of the filters, the typography, spacing, and colors should match the designs

**Review by whamester — CHANGES_REQUESTED** [review]
Clear the filters: This should be a two-step action: clear the filters and then apply the changes.

- Clicking the Clear All should clear the selected checkboxes and display the count of the Show # reports button, by clicking on Show # reports is when the filters are applied on the map.


Styles:

Double-check the styles of the filters, the typography, spacing, and colors should match the designs

**Review by whamester — APPROVED** [review]

**Pull request merged**